packages/sampler/stories/qa/Header.js

```diff
@@ -16,12 +16,13 @@ const inputData = {
 	shortTitle: 'Enact',
 	shortTitleBelow: 'An app framework',
 	shortSubTitleBelow: 'Built atop React',
-	shortRtlTitle: 'כתוביות למט',
+	shortRtlTitle: 'כתוביות למט ',
```

Comment:
Is this space addition intentional?

Reply:
Extra space was not intended. Thank you for catching it. Corrected.